packages/blockchain-link/src/types/blockbook/common.ts

```diff
@@ -0,0 +1,11 @@
+export interface VinVout {
```

Comment:
i've got better idea for that:
https://github.com/trezor/trezor-suite/pull/3474/commits/ad9abe554a19d2c6daeb53d6b4e40d1a2f3ac5ee

Reply:
Rebased on develop and reverted my changes in blockchain-link.